cfr-heartbeat.yaml

```diff
@@ -0,0 +1,21 @@
+- id: HEARTBEAT_TACTIC_2_75
+  template: cfr_urlbar_chiclet
+  content:
+    layout: chiclet_open_url
+    category: cfrHeartbeat
+    bucket_id: HEARTBEAT_TACTIC_2_75,
+    notification_text: Help improve Firefox
+    active_color: '#595e91'
+    action:
+      url: https://qsurvey.mozilla.com/s3/goals-survey
+      where: tabshifted
+  priority: 4
```

Comment:
What's the rationale for priority?

Reply:
This is to force the message before the Tracking Protection messages for new profiles (not to get stuck on the provider freq cap). I realize it's not needed because this targeting is less restrictive and will trigger first anyway.